internal/agent/agent.go

```diff
@@ -153,8 +147,13 @@ func New(conf *Config) (*Agent, error) {
 	} else {
 		// Didn't detect the redis container, starting a new one...
 		log.Info().Msg("No redis container detected, creating a new one")
-		container = docker.NewContainer(docker.ContainerConfig{
-			Image:     "redis:7.0.4",
+		container = virtual.NewContainer(virtual.ContainerConfig{
+			Image: "redislabs/rejson:2.4.1",
+			// Cmd: []string{
+			// 	"/data/redis.conf",
+			// 	"--loadmodule",
+			// 	"/usr/lib/redis/modules/rejson.so",
+			// },
```

Comment:
```suggestion
```
Don't comment out commented lines, this is one way to expose secrets by accident :)